spec/features/note_spec.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+require 'rails_helper'
+require 'capybara/rspec'
+include RequestHelpers
+
+feature 'Team leader creates a new note' do
+  let(:user){ create_logged_in_user }
+  let(:member) { create(:user, email: "a@gmail.com", team_leader: user) }
+  let(:initiative) { create(:initiative, user: member) }
+
+  scenario 'clicks add note button' do
+    visit user_initiative_path(member, initiative )
+    expect(page).to have_button("Add Note")
+    fill_in 'note_body', with: 'Note test'
+    click_button 'Add Note'
+    expect(page).to have_content('Note test')
+  end
+end
+
+feature 'Team leader creates a new note' do
+  let(:user){ create_logged_in_user }
+  let(:member) { create(:user, email: "a@gmail.com", team_leader: user) }
+  let(:initiative) { create(:initiative, user: member) }
+
+  scenario 'clicks back button' do
+    visit user_initiative_path(member, initiative )
+    click_link 'Back'
+    expect(page).to have_content("#{member.fullname}'s Initiatives")
```

Comment:
Does this text disappear in the process of adding a note? If not, how do you know if your action worked? You might need to be looking for something different, maybe a button?

Reply:
changed the test to button